src/js/visual/configurator-csr.js

```diff
@@ -1759,7 +1774,7 @@ ripe.ConfiguratorCsr.prototype._onPostConfig = async function(self, config) {
 /**
  * @ignore
  */
-ripe.ConfiguratorCsr.prototype._registerHandlers = function() {
+ripe.ConfiguratorCsr.prototype._registerGlobalHandlers = function() {

```

Comment:
These first started as just the only existing handlers so the name made sense, but now the name is too general so if you are renaming it I I woul probably rename this to `_registerElementHandlers()`,  `_registerDomHandlers()` or something like that.